microcosm_flask/matchers.py

```diff
@@ -63,3 +67,30 @@ def schema_class(self):
 
     def describe_to(self, description):
         description.append_text("expected {}".format(prettify(self.expected)))
+
+
+class URIMatcher(BaseMatcher):
```

Comment:
The previous `URLMatcher` (name changed  here) was in the `test` tree so it couldn't be imported in other projects. (We may not need to use it BTW if we normalize URIs on the input side via the new field)